vw_golf_mk4.dbc

```diff
@@ -1074,41 +1074,38 @@ BO_ 1550 Einheiten_1: 2 XXX
  SG_ MFA_v_Einheit_02 : 0|1@1+ (1,0) [0|1] "" XXX
 
 BO_ 872 ACC_System: 8 XXX
- SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
- SG_ COUNTER : 8|4@1+ (1,0) [0|15] "" XXX
+ SG_ ACS_Checksum : 0|8@1+ (1,0) [0|1] "" XXX
+ SG_ ACS_Zaehler : 8|4@1+ (1,0) [0|1] "" XXX
```

Comment:
We use all capital names for these so the parser can recognize which signals are counters/checksums